flink-runtime/src/main/java/org/apache/flink/runtime/rest/messages/MessageHeaders.java

```diff
@@ -63,4 +63,13 @@
 	 * @return description for the header
 	 */
 	String getDescription();
+
+	/**
+	 * Returns whether this header allows file uploads.
+	 *
+	 * @return whether this header allows file uploads
+	 */
+	default boolean acceptsFileUploads() {
+		return false;
+	}
```

Comment:
Should this maybe go into `UntypedResponseMessageHeaders`? At the moment one can upload files for a `AbstractHandler` (e.g. `AbstractTaskManagerFileHandler`) implementation and also has access to it via the `HandlerRequest` without being able to specify whether file upload is allowed or not.